src/api/internal.rs

```diff
@@ -587,21 +605,24 @@ impl<T: Pixel> ContextInner<T> {
 
     #[cfg(feature = "dump_lookahead_data")]
     {
+      let data_location = Self::build_dump_properties();
       let plane = &fs.input_qres;
+      let mut file_name = format!("{:010}-qres", fi.input_frameno);
       image::GrayImage::from_fn(
         plane.cfg.width as u32,
         plane.cfg.height as u32,
         |x, y| image::Luma([plane.p(x as usize, y as usize).as_()]),
       )
-      .save(format!("{}-qres.png", fi.input_frameno))
+      .save(Path::new(&data_location).join(file_name).with_extension("png"))
```

Comment:
PathBuf has a [join](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.join).

Reply:
Fixed, for a moment, I did not think that way :) 
Thanks